runtime/gc_structs/VMThreadStackSlotIterator.cpp

```diff
@@ -63,6 +63,41 @@ vmThreadStackFrameIterator(J9VMThread * currentThread, J9StackWalkState * walkSt
 
 } /* extern "C" */
 
+void
```

Comment:
Either make this is a static method or make the name of the method more specific for GC purposes. In theory there could be a same/similar method somewhere in core VM code.

Reply:
as I said in hpp comment, make this method a class member and remove 'forGC'